common/src/stack/command/stack/commands/__init__.py

```diff
@@ -473,7 +473,9 @@ def getPallets(self, args, params):
 			""", (arg, version, rel, arch, os))
 
 			if not rows and arg != '%':
-				raise ArgNotFound(self, arg, 'pallet')
+				# Output a string of command line parameters and its values 
+				param_errors = ','.join(f'{key}:{value}' for key, value in params.items())
```

Comment:
This string builder should probably live in the Exception, not the ArgProcessor.

Also the error string should look the same as it was input - that is instead of `k:v,k2:v2`, `k=v k2=v2`.